pkg/webhook/package_webhook.go

```diff
@@ -93,6 +93,10 @@ func (v *packageValidator) isPackageConfigValid(p *v1alpha1.Package, activeBundl
 		return false, fmt.Errorf("package %s does not contain any versions", p.Name)
 	}
 
+	if packageInBundle.WorkloadOnly && !p.IsInstalledOnWorkload() {
```

Comment:
This won't be a blocker for this PR, but I just wonder - where do you plan to define if a package is WorkloadOnly or not and have that be in place yet?

Reply:
Another PR is coming through for that.